### PR TITLE
End to end selenium tests (Stripe and Paypal)

### PR DESCRIPTION
**Pull request by jranks123**
This PR adds selenium end to end tests for the Stripe and Paypal checkout flows. 

The tests can be run locally with the command `sbt test-only *StripeCheckoutSpec*

### CODE REVIEW

File: test/acceptance/StripeCheckoutSpec.scala [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  val contributionAmount = pages.ContributionAmount
+  val yourDetails = pages.YourDetails
+  val stripeCheckout = pages.StripeCheckout
+  val paypalCheckout = pages.PaypalCheckout
+  val thankYou = pages.ThankYou
+  val postPayment = pages.PostPayment
+
+
+    "The OneBrowserPerTest trait" must {
```
Comment (desbo) [inline test/acceptance/StripeCheckoutSpec.scala]: indentation is a little wild in this file

**Comment by michaelwmcnamara** [comment]
just tried locally - you need a local instance of contributions-frontend up and running.
Once this is up and running, if you run the command "sbt test-only StripeCheckoutSpec.scala":
All tests seem to run and pass, however there is an error message at the end of the run:

"[error] No valid parser available.
[error] StripeCheckoutSpec.scala"

**Comment by michaelwmcnamara** [comment]
Added background setup copied from membership-frontend. This will allow us to use the same method as they do to run our tests - specifically a hook from prout to Travis, which then runs the tests on saucelabs.

**Review by desbo — APPROVED** [review]

**Review by joelochlann — APPROVED** [review]
Looks good to me

**Pull request merged**